freeipa/src/main/java/com/sequenceiq/freeipa/service/upgrade/ccm/CcmParametersConfigService.java

```diff
@@ -25,6 +26,9 @@ public class CcmParametersConfigService {
 
     private static final String UPGRADE_CCM_SLS_PATH = "/upgradeccm/init.sls";
 
+    @Value("${ccmRevertJob.activationInMinutes:5}")
```

Comment:
please move the deafult value to appl yaml

Reply:
ok